priv/repo/migrations/20170308222552_create_auth_token.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateAuthToken do
+  use Ecto.Migration
+
+  def change do
+    create table(:auth_token) do
```

Comment:
@snewcomer I see that @begedin intends this to be used for potentially passwordless authentication (i.e. the Slack "magic link") in addition to password reset. I'd just document that in the `@moduledoc` for this, then.